libpod/runtime_img.go

```diff
@@ -408,6 +419,27 @@ func (k *Image) Pull(writer io.Writer) error {
 	return nil
 }
 
+// Remove calls into container storage and deletes the image
+func (k *Image) Remove(force bool) (string, error) {
+	if k.LocalName == "" {
+		// This populates the images local name
+		_, err := k.GetLocalImageName()
+		if err != nil {
+			return "", errors.Wrapf(err, "unable to remove %s", k.Name)
```

Comment:
Can this be a slightly more descriptive error?

Reply:
yup